pyravendb/store/document_session.py

```diff
@@ -19,6 +20,129 @@ def __init__(self, commands, deferred_command_count, entities=None):
         self.deferred_command_count = deferred_command_count
 
 
+class _RefEq:
+    def __init__(self, ref):
+        if isinstance(ref, _RefEq):
+            self.ref = ref.ref
+        self.ref = ref
+
+    def __eq__(self, other):
+        return (
```

Comment:
shouldn't we base `__eq__()` on whether we're dealing with `_RefEq` instance or not by making a check `isinstance(ref, _RefEq)` like in `__init__()` ?

Reply:
Adjusted the concept little bit. Left a comment above __eq__ in the latest [commit](https://github.com/ravendb/ravendb-python-client/pull/101/commits/e36398ab17903a6171a2a3bfff772f52f751fc18#diff-f0339370256b6c60b83d633107743b0ac1d35a15d6fee40404b3d2183141089bR30).